static/js/services/db.js

```diff
@@ -21,6 +22,17 @@ angular.module('inboxServices').factory('DB',
 
     $window.PouchDB.adapter('worker', require('worker-pouch/client'));
 
+    var getUsername = function(remote) {
+      var username = Session.userCtx().name;
+      if (!remote) {
+        return username;
+      }
+      // escape username in case they user invalid characters
+      return username.replace(DB_NAME_BLACKLIST, function(match) {
+        return '(' + match.charCodeAt(0) + ')';
```

Comment:
Yeah that looks better - I didn't even know it was there. But it's also not supported in our version of Chrome.